client/my-sites/site-settings/settings-performance/index.js

```diff
@@ -0,0 +1,25 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import { makeLayout, render as clientRender } from 'controller';
+import { navigation, siteSelection } from 'my-sites/controller';
+import { performance } from './controller';
+import { siteSettings } from 'my-sites/site-settings/settings-controller';
```

Comment:
I think I misleaded you here, assuming that the settings controller exports functions. But it doesn't: https://github.com/Automattic/wp-calypso/blob/7b91038e6b42d9fc63722cdc051d76e95e5ba698/client/my-sites/site-settings/settings-controller.js#L19

To fix this, we have to change this import back to:

```
import settingsController from 'my-sites/site-settings/settings-controller';
```

and refer to the controller as `settingsController.siteSettings` again.